pulp_python/app/serializers.py

```diff
@@ -164,19 +164,16 @@ class Meta:
 class PythonImporterSerializer(platform.ImporterSerializer):
     """
     A Serializer for PythonImporter.
-
-    Add any new fields if defined on PythonImporter.
-    Similar to the example above, in PythonContentSerializer.
-    Additional validators can be added to the parent validators list
-
-    For example::
-
-    class Meta:
-        validators = platform.ImporterSerializer.Meta.validators + [myValidator1, myValidator2]
     """
 
+    project_names = serializers.ListField(
```

Comment:
what do you think of calling this `requirements`? For now, just a list of project names, but in the future it could take specific versions or the other syntax in requirements.txt